radio/src/translations.cpp

```diff
@@ -1,714 +1,725 @@
-/*

```

Comment:
Oh ok, it's because I used a script to modify all licenses in one shot
Is it possible you don't change it? I mean, you only add in this commit the real changes, another commit will follow with unification of all line endings